src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -734,13 +754,45 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
     };
     let secrets_reader = secrets_controller.reader();
     let now = SYSTEM_TIME.clone();
-    let persist_clients = PersistClientCache::new(
-        PersistConfig::new(&mz_environmentd::BUILD_INFO, now.clone()),
-        &metrics_registry,
+
+    let persist_pubsub_addr = format!(
+        "http://{}:{}",
+        args.persist_pubsub_hostname,
```

Comment:
The hostname field is needed to advertise to other clients the routable host for this listener?

Reply:
yep! it'll be provided by the `environment-controller` and likely backed by a Kube Service